entries/i/id.muni.cz.json

```
@@ -0,0 +1,18 @@
+{
+  "MUNI Unified Login": {
+    "domain": "id.muni.cz",
+    "additional-domains": [
+      "ics.id.muni.cz"
+    ],
+    "tfa": [
+      "totp",
+      "u2f"
+    ],
+    "documentation": "https://it.muni.cz/en/services/jednotne-prihlaseni-na-muni/navody/how-to-set-up-multi-factor-authentication",
+    "recovery": "https://it.muni.cz/en/forms/report-a-problem-with-single-sign-on",
```

A region element would be appropriate here.
```suggestion
    "recovery": "https://it.muni.cz/en/forms/report-a-problem-with-single-sign-on",
    "regions": [
      "cz"
    ],
```

---

[CONTRIBUTING.md](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md) says:
```
If a site (with or without 2FA) is only available in certain countries - for example a government site - you can note this with the regions field
```
but the site is available to people world wide and provides access to services from many countries. Should I still add the region?

---

Yes please.